src/BTVNanoCommissioning/workflows/ctag_DY_valid_sf.py

```diff
@@ -268,92 +274,95 @@ def process_shift(self, events, shift_name):
                 if syst == "nominal" or syst == shift_name
                 else weights.weight(modifier=syst)
             )
-            for histname, h in output.items():
-                if (
-                    "Deep" in histname
-                    and "btag" not in histname
-                    and histname in events.Jet.fields
-                ):
-                    h.fill(
-                        syst,
-                        genflavor,
-                        sel_jet[histname],
-                        weight=weights.partial_weight(exclude=exclude_btv),
-                    )
-                elif (
-                    "PFCands" in events.fields
-                    and "PFCands" in histname
-                    and histname.split("_")[1] in events.PFCands.fields
-                ):
-                    h.fill(
-                        syst,
-                        flatten(ak.broadcast_arrays(genflavor, spfcands["pt"])[0]),
-                        flatten(spfcands[histname.replace("PFCands_", "")]),
-                        weight=flatten(
-                            ak.broadcast_arrays(
-                                weights.partial_weight(exclude=exclude_btv),
-                                spfcands["pt"],
-                            )[0]
-                        ),
-                    )
-                elif (
-                    "posl_" in histname
-                    and histname.replace("posl_", "") in sposmu.fields
-                ):
-                    h.fill(
-                        syst,
-                        flatten(sposmu[histname.replace("posl_", "")]),
-                        weight=weight,
-                    )
-                elif (
-                    "negl_" in histname
-                    and histname.replace("negl_", "") in snegmu.fields
-                ):
-                    h.fill(
-                        syst,
-                        flatten(snegmu[histname.replace("negl_", "")]),
-                        weight=weight,
-                    )
+            # for histname, h in output.items():
```

Comment:
@Ming-Yan  why has this code been commented out?

Reply:
sorry this is for other checks, revert it back to previous version